main.go

```diff
@@ -126,73 +109,266 @@ func (e *periodicExporter) Collect(ch chan<- prometheus.Metric) {
+	e.gauge(metricsChan, labels, labelValues, "mesos_disk_used", float64(stats.DiskUsed), "Disk used")
+
+	e.gauge(metricsChan, labels, labelValues, "mesos_executors_registering", float64(stats.ExecutorsRegistering), "Executors registering")
+	e.gauge(metricsChan, labels, labelValues, "mesos_executors_running", float64(stats.ExecutorsRunning), "Executors running")
+	e.counter(metricsChan, labels, labelValues, "mesos_executors_terminated", float64(stats.ExecutorsTerminated), "Executors terminated")
+	e.gauge(metricsChan, labels, labelValues, "mesos_executors_terminating", float64(stats.ExecutorsTerminating), "Executors terminating")
+
+	e.gauge(metricsChan, labels, labelValues, "mesos_frameworks_active", float64(stats.FrameworksActive), "Frameworks active")
+	e.counter(metricsChan, labels, labelValues, "mesos_invalid_framework_messages", float64(stats.InvalidFrameworkMessages), "Invalid framework messages")
+	e.counter(metricsChan, labels, labelValues, "mesos_invalid_status_updates", float64(stats.InvalidStatusUpdates), "Invalid status updates")
+
+	e.gauge(metricsChan, labels, labelValues, "mesos_mem_percent", stats.MemoryPercentage, "Memory percentage")
+	e.gauge(metricsChan, labels, labelValues, "mesos_mem_total", float64(stats.MemoryTotal), "Memory total")
+	e.gauge(metricsChan, labels, labelValues, "mesos_mem_used", float64(stats.MemoryUsed), "Memory used")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_recovery_errors", float64(stats.RecoveryErrors), "Recovery errors")
+	e.gauge(metricsChan, labels, labelValues, "mesos_registered", float64(stats.Registered), "Registered")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_tasks_failed", float64(stats.TasksFailed), "Failed tasks")
+	e.counter(metricsChan, labels, labelValues, "mesos_tasks_finished", float64(stats.TasksFinished), "Finished tasks")
+	e.counter(metricsChan, labels, labelValues, "mesos_tasks_killed", float64(stats.TasksKilled), "Killed tasks")
+	e.counter(metricsChan, labels, labelValues, "mesos_tasks_lost", float64(stats.TasksLost), "Lost tasks")
+	e.gauge(metricsChan, labels, labelValues, "mesos_tasks_running", float64(stats.TasksRunning), "Running tasks")
+	e.gauge(metricsChan, labels, labelValues, "mesos_tasks_staging", float64(stats.TasksStaging), "Staging tasks")
+	e.gauge(metricsChan, labels, labelValues, "mesos_tasks_starting", float64(stats.TasksStarting), "Starting tasks")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_uptime_secs", stats.UptimeSecs, "Uptime in seconds")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_valid_framework_messages", float64(stats.ValidFrameworkMessages), "Valid framework messages")
+	e.counter(metricsChan, labels, labelValues, "mesos_valid_status_updates", float64(stats.ValidStatusUpdates), "Valid status updates")
+
+	e.gauge(metricsChan, labels, labelValues, "mesos_system_cpus_total", float64(stats.SystemCpuTotal), "System CPU total")
```

Comment:
Ah, thanks for the heads-up. Haven't seen https://github.com/prometheus/node_exporter before, will check that out also